mount/mount_windows.go

```diff
@@ -166,6 +185,18 @@ func removeSMBMapping(remotepath string) (string, error) {
 func (mounter *Mounter) Unmount(target string) error {
 	klog.V(4).Infof("azureMount: Unmount target (%q)", target)
 	target = NormalizeWindowsPath(target)
+	source, err := os.Readlink(target)
+	if err != nil {
+		klog.Infof("failed to get source from target %q, %v", target, err)
+		return err
+	}
+	output, err := exec.Command("powershell", "/c", "net use", source, "/delete").CombinedOutput()
+	if err != nil {
+		klog.Infof("failed to delete connection from source %q %v", source, err)
```

Comment:
Can we return this message as the error and let the caller log?

Reply:
removed